cognite/dm_clients/domain_modeling/relationship_api.py

```diff
@@ -36,9 +36,27 @@ def __init__(
         self.schema_version = schema_version
         self.space_id = space_id
 
+    def add(self, attribute: str, start_ext_id: str, end_ext_ids: Iterable[str]) -> None:
```

Comment:
I think this need to be apply to be consistent with the underlying endpoint.

Reply:
We already have `apply`, this is a distinct operation.